locales/fr_FR/pcsx2_Main.po

```diff
@@ -2239,11 +2235,8 @@ msgstr "Précharger le hack TLB pour éviter un manque de TLB dans Goemon"
 
 #: pcsx2/gui/Panels/GameFixesPanel.cpp:108
 #, fuzzy
-msgid ""
-"VU I bit Hack avoid constant recompilation (Scarface The World Is Yours)"
-msgstr ""
-"Hack bit VU I: éviter une recompilation constante (Scarface The World is "
-"Yours)"
+msgid "VU I bit Hack avoid constant recompilation (Scarface The World Is Yours)"
+msgstr "Hack bit VU I: éviter une recompilation constante (Scarface The World is Yours)"
```

Comment:
je traduirai `constant` par `permanente`

Reply:
Oui, ceci semble plus approprié.
"Yes, it is more correct"